docs_outdoornav_user_manual/release_notes.mdx

```diff
@@ -6,6 +6,37 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
+## 0.10.0
+
+### New Features
+
+-   Added the ability to start mission from a specific Waypoint
+-   Added the ability to resume mission from current location
+-   Added the option to include on-start and on-stop tasks to a Mission
+-   Added the option to continue a Mission if a Task should fail
+-   Full release of the API examples repository to the public (link API examples top level page)
+
+### API Features
+
+-   Action definition changes (see [API Endpoints](./api/api_endpoints/autonomy_api))
+    -   Mission.action added from_start and *start_waypoint_uuid* fields
+    -   ExecuteMissionByUuid.action:  added from_start and *start_waypoint_uuid* fields
+-   Message Definition changes
+    -   Mission.msg: added an array of on_start and and array of on_stop tasks fields
+    -   Task.msg: added a boolean allow_failure field
+-   Updated /safety/safety_stop message type (link to Safety message definition ./definitions.mdx#msg-safety)

```

Comment:
Is this supposed to be a hyperlink?

Reply:
It is yes. This was an invalid link but it should be fixed now.